src/lidar_tools/pdal_pipeline.py

```diff
@@ -102,13 +98,13 @@ def create_dsm(extent_geojson: str, # processing_extent.geojson
     intensity_fn_list = []
     for i, reader in enumerate(readers):
         print(f"Processing reader #{i}")
-        dsm_file = f'{output_path}/{prefix}_dsm_tile_aoi_{str(i).zfill(4)}.tif'
-        dtm_file = f'{output_path}/{prefix}_dtm_tile_aoi_{str(i).zfill(4)}.tif'
-        intensity_file = f'{output_path}/{prefix}_intensity_tile_aoi_{str(i).zfill(4)}.tif'
+        dsm_file = output_path / f'{prefix}_dsm_tile_aoi_{str(i).zfill(4)}.tif'
+        dtm_file = output_path / f'{prefix}_dtm_tile_aoi_{str(i).zfill(4)}.tif'
+        intensity_file = output_path / f'{prefix}_intensity_tile_aoi_{str(i).zfill(4)}.tif'
```

Comment:
@ShashankBice I realized my recommended change on the last PR messed up the paths. So here I've switched to only use pathlib instead of a mix of os.path and pathlib